tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
 	"compilerOptions": {
+		"allowSyntheticDefaultImports": true,
```

Comment:
I figured out why this wasn't working earlier today...

1. With the current ESBuild settings, `mustache` does not support _namespace_ imports (`import * as Mustache from "mustache"`), just _default_ imports (`import Mustache from "mustache";`)
2. The `@types/mustache` `.d.ts` typings for the `mustache` module declare it as exporting things in the namespace style (e.g. `export const name: string;`, `export as namespace Mustache;`)
3. Without this TSConfig change, TypeScript was giving a type error when we tried to change the import to be a _default_ import -- because the typings told it that's not supported
4. [`allowSyntheticDefaultImports`](https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports) is a TSConfig compiler option that tells TypeScript to assume _default_ imports are supported regardless of the typings

Overall, ESBuild does take in all input files imported by `src/content-script.ts` and bundle them into `lib/content-script.js`. We know this works because there are already multiple `import`s at the top of the file! Just, this new `mustache` import was giving grief because the type system understanding of the code didn't match the actual runtime values.